src/lib.rs

```diff
@@ -272,7 +272,7 @@ pub fn parse_path(path: &PathBuf, config: ParseConfig) -> anyhow::Result<ParseOu
     let index_context = IndexContext {
         css: CSS,
         directory: directory
-            .drain()
+            .drain(..)
```

Comment:
What does this syntax mean?

Reply:
.. apparently means "the entire thing", IDK man, I just read the docstring for drain haha